<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>exportLoading</class>
 <widget class="QDialog" name="exportLoading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>220</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>220</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>220</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Exporting</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QFrame" name="loadingFrame">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(243, 243, 243);
	border: 1px solid rgb(100, 100, 100);
border-radius: 5px;
}

QLabel{
border: none;
	color: rgb(100, 100, 100);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="gifPlaceholder">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>130</width>
        <height>130</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>130</width>
        <height>130</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>130</width>
        <height>130</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="loadingMsg">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>211</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>sample message . . .</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
